digital_land/phase/organisation.py

```diff
@@ -6,11 +6,21 @@ class OrganisationPhase(Phase):
     lookup the organisation
     """
 
-    def __init__(self, organisation={}):
+    def __init__(self, issues=None, organisation={}):
```

Comment:
Add this at the end after organisation, it will stop any errors from mixed up order happening